bundles/org.openhab.binding.shelly/README.md

```diff
@@ -312,26 +289,23 @@ A new alarm will be triggered on a new condition or every 5 minutes if the condi
 |TEMP_OVER   |Above "temperature over" threshold                                                                               |
 
 
-```
-rule "Shelly Alarm"
-when
-    Channel "shelly:shelly1:XXXXXX:device#alarm" triggered
-then
-    logInfo("Shelly", "n alarm condition was detected:" + receivedEvent.toString())
-end
-```
+Refer to section Full Example:.rules for examples how to catch alarm triggers in openHAB rules
```

Comment:
```suggestion
Refer to section [Full Example/.rules](#shelly.rules) for examples how to catch alarm triggers in openHAB rules
```

Reply:
Are you sure that this is the connect syntax? If I include the # for shelly.rules Eclipse displays an error
Now I have '[Full Example:shelly.rules](shelly.rules) ', but not if the link works

Reply:
found it, '.' has to be replaced by '-' in the anchor name

Reply:
I tested it with Visual Studio Code and only a dot worked. I was surprised as I first tried a dash, too. We'll see what's correct.